### PR TITLE
Tenant metrics

### PR DESCRIPTION
**Pull request by gouthamve**
Added the easiest write metrics for #103

### CODE REVIEW

File: pkg/distributor/distributor.go [hunk 1]
```diff
@@ -19,29 +20,34 @@ import (
...
-func init() {
-	prometheus.MustRegister(sendDuration)
-	prometheus.MustRegister(ingesterAppends)
-	prometheus.MustRegister(ingesterAppendFailures)
-}
+	bytesIngested = promauto.NewCounterVec(prometheus.CounterOpts{
+		Namespace: "loki",
+		Name:      "distributor_bytes_received_total",
+		Help:      "The total number of uncompressed bytes received per user",
+	}, []string{"org"})
```
Comment (tomwilkie) [inline pkg/distributor/distributor.go]: In cortex we used "user"[0]; in Loki we seem to be converging on the terms "instance" and "tenant".

[0] https://github.com/cortexproject/cortex/blob/master/pkg/distributor/distributor.go#L50

Comment (gouthamve) [reply pkg/distributor/distributor.go]: Should I just run with tenant then? See: https://github.com/grafana/loki/issues/131

Comment (tomwilkie) [reply pkg/distributor/distributor.go]: Instance is preferred I'd say; tenant is used in docs/presentations, but thats about it.

**Comment by gouthamve** [comment]
@tomwilkie PTAL

**Comment by negbie** [comment]
@gouthamve 

shouldn't
bytesIngested.WithLabelValues(userID).Add(float64(bytesCount))
inesIngested.WithLabelValues(userID).Add(float64(lineCount))

be
bytesIngested.WithLabelValues(ingester.Addr).Add(float64(bytesCount))
inesIngested.WithLabelValues(ingester.Addr).Add(float64(lineCount))

**Comment by negbie** [comment]
You have to move the metrics to func sendSamplesErr. One advantage would be there is already a loop over req.Streams.

**Comment by gouthamve** [comment]
Ah, we want to track the lines ingested per user and not per ingester. And when Prometheus scrapes these metrics, it automatically adds the ingester address as a labels so we can see how much each ingester is getting too.

**Pull request merged**